modules/auxiliary/admin/http/manage_engine_dc_create_admin.rb

```diff
@@ -62,7 +62,7 @@ def run
       }
     })
 
-    # Yes, "sucess" is really mispelt, as is "Servelet" ... !
+    # Yes, "success" is really misspelt, as is "Servelet" ... !
```

Comment:
Needs reverted

Reply:
when its all spelled out like that yea, good catch!